docs/edge_migration.md

```diff
@@ -0,0 +1,73 @@
+# Plans for migrating edge traffic to GCP Ingestion
+
+This document outlines plans to migrate edge traffic to go through the code in this repository.
```

Comment:
Nit: migration happens from one place to another. I'd suggest "migrate edge traffic from AWS to GCP" or similar.